test/Middleware/ImplicitOptionsMiddlewareTest.php

```diff
@@ -73,12 +73,11 @@ public function testReturnsResultOfHandlerWhenRouteSupportsOptionsExplicitly()
     {
         $route = $this->prophesize(Route::class);
 
-        $result = $this->prophesize(RouteResult::class);
-        $result->getMatchedRoute()->will([$route, 'reveal']);
+        $result = RouteResult::fromRoute($route->reveal());
```

Comment:
Changed here to **not** mocking the RouteResult because if we use more methods we need always extend our mock.
And what's more we can easily mock `RouteResult` wrong way. We can create new instance only by using `fromRoute` or `fromRouteFailure` but we can create invalid mock as we have to mock 4-5 public methods...